### PR TITLE
chore: ha-node nvme connect over uds to csi-node

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
Signed-off-by: Abhinandan Purkait <purkaitabhinandan@gmail.com>

### CODE REVIEW

File: chart/templates/mayastor/agents/ha/ha-node-daemonset.yaml [hunk 1]
```diff
@@ -70,6 +71,8 @@ spec:
           mountPath: /sys
         - name: run-udev
           mountPath: /run/udev
+        - name: plugin-dir
+          mountPath: /csi
```
Comment (tiagolobocastro) [inline chart/templates/mayastor/agents/ha/ha-node-daemonset.yaml:75]: hmm is there any point for `Values.csi.node.socketPath` if `/csi` is still hardcoded? Or should we make `/csi` part of the variables as well and derive socketPath?

Comment (Abhinandan-Purkait) [reply chart/templates/mayastor/agents/ha/ha-node-daemonset.yaml:75]: Yeah, was also thinking of same. That would be better.

Comment (Abhinandan-Purkait) [reply chart/templates/mayastor/agents/ha/ha-node-daemonset.yaml]: Done.

**Review by tiagolobocastro — APPROVED** [review]

**Review by abhilashshetty04 — APPROVED** [review]

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**